### PR TITLE
Fix pages being called lines in BookScreen

### PR DESCRIPTION
**Pull request by Juuxel**
`BookScreen.getLines(CompoundTag)` actually gets the book tag's list of pages.

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-		METHOD method_17561 getLine (I)Lnet/minecraft/class_2561;
-			ARG 1 line
+		METHOD method_17560 getPageCount ()I
+		METHOD method_17561 getPage (I)Lnet/minecraft/class_2561;
+			ARG 1 index
 		METHOD method_17562 create (Lnet/minecraft/class_1799;)Lnet/minecraft/class_3872$class_3931;
 			ARG 0 stack
-		METHOD method_17563 getLineOrDefault (I)Lnet/minecraft/class_2561;
-			ARG 1 line
+		METHOD method_17563 getPageOrDefault (I)Lnet/minecraft/class_2561;
```
Comment (liach) [inline mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping]: `getPage`

File: mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping [hunk 2]
```diff
@@ -1,18 +1,24 @@
 CLASS net/minecraft/class_3872 net/minecraft/client/gui/screen/ingame/BookScreen
 	CLASS class_3931 Contents
-		METHOD method_17560 getLineCount ()I
-		METHOD method_17561 getLine (I)Lnet/minecraft/class_2561;
-			ARG 1 line
+		METHOD method_17560 getPageCount ()I
+		METHOD method_17561 getPage (I)Lnet/minecraft/class_2561;
```
Comment (liach) [inline mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping]: `getPageUnchecked`

Comment (Gegy) [reply mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping]: Does unchecked imply a crash if invalid? Or null return?

Comment (liach) [reply mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping]: unchecked means like getting something unsafe, like nullable

Comment (Gegy) [reply mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping]: The null value isn't inherently unsafe -- only when it is dereferenced.

Comment (liach) [reply mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping]: the unchecked method here may throw index out of bounds exception and is impl only; all calls actually go through `getPageOrDefault`, which imo should be renamed to `getPage`

File: mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping [hunk 3]
```diff
@@ -42,7 +48,8 @@ CLASS net/minecraft/class_3872 net/minecraft/client/gui/screen/ingame/BookScreen
 	METHOD method_17059 updatePageButtons ()V
 	METHOD method_17554 setPageProvider (Lnet/minecraft/class_3872$class_3931;)V
 		ARG 1 pageProvider
-	METHOD method_17555 getLines (Lnet/minecraft/class_2487;)Ljava/util/List;
+	METHOD method_17555 getPages (Lnet/minecraft/class_2487;)Ljava/util/List;
```
Comment (Gegy) [inline mappings/net/minecraft/client/gui/screen/ingame/BookScreen.mapping]: Should this be `parsePages` / `readPages` since it is parsing from nbt?

**Comment by Juuxel** [comment]
All suggestions done.

**Review by Gegy — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**